### ISSUE TITLE
Add API to nullify log_data

### ISSUE THREAD

**Issue by palkan** [palkan/logidze#108]
Sometimes it might be necessary to _reset_ the history for a record (or records) (e.g. with some GDPR-related stuff).

Let's add an API to do that. Something like this:

```ruby
# for single record
record.reset_log_data #=> which is, probably, equal to Logidze.without_logging { record.update_column(:log_data, nil) }

# for relation
User.where(...).reset_log_data #=> Logidze.without_logging { relation.update_all(log_data: nil) }
```

**Comment by Arkweid** [comment]
Take this one

**Comment by Arkweid** [comment]
@palkan opened: #110

**Comment by palkan** [comment]
Closed by #110
